xdpderp/headers/update.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package main
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const (
+	libbpfVersion = "1.4.3"
+	prefix        = "libbpf-" + libbpfVersion
+)
+
+var (
+	filesToExtract = map[string]struct{}{
+		prefix + "/LICENSE.BSD-2-Clause":  {},
+		prefix + "/src/bpf_endian.h":      {},
+		prefix + "/src/bpf_helper_defs.h": {},
+		prefix + "/src/bpf_helpers.h":     {},
+		prefix + "/src/bpf_tracing.h":     {},
+	}
+)
+
+var (
+	flagDest = flag.String("dest", ".", "destination directory")
+)
+
+func main() {
+	flag.Parse()
+
+	f, err := os.CreateTemp(os.TempDir(), "libbpf")
```

Comment:
`CreateTemp` docs suggest `""` as the dir if you want the default

Reply:
updated in https://github.com/tailscale/tailscale/pull/12390/commits/fb519d6b923b22517050f931bf1541df4079e92b